### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by matkoch**
# Types of Changes

## Prerequisites

Please make sure you can check the following two boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

## Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

## Description

Just a couple of fixes. Let me know if you want them to be split into several commits like "interpunction" and "actual typos".

### CODE REVIEW

File: docs/general/01-Basics.md [hunk 1]
```diff
@@ -120,15 +120,15 @@ This works and directly returns an object of type `IHtmlParagraphElement`. No ca
 
 ### Beyond the DOM
 
-AngleSharp provides several properties and methods that are not accessible via the standardized DOM properties and methods. To distinguish between standardized and extended a simple attribute class called `DomNameAttribute` has been added. The attribute is applied in those cases, where the decorated class / event / method or property is also specified in the official W3C standard. Additionally the official name is set, since AngleSharp follows the PascalCase convention, while the DOM follows the camelCase convention.
+AngleSharp provides several properties and methods that are not accessible via the standardized DOM properties and methods. To distinguish between standardized and extended a simple attribute class called `DomNameAttribute` has been added. The attribute is applied in those cases, where the decorated class / event / method or property is also specified in the official W3C standard. Additionally, the official name is set, since AngleSharp follows the PascalCase convention, while the DOM follows the camel-case convention.
```
Comment (FlorianRappl) [inline docs/general/01-Basics.md]: Here it was intentional (see `PascalCase` - the casing of `camelCase` already transports an example for the convention).

**Review by FlorianRappl — APPROVED** [review]
Just great - I'd just keep the `camelCase`.

**Comment by matkoch** [comment]
Thanks for all the merges! :)

**Comment by FlorianRappl** [comment]
Thanks for all the awesome contributions! :beers:

**Pull request merged**